docs/cep/sink/sink-types/stream-sink.md

```diff
@@ -22,11 +22,19 @@ The stream sink syntax is as follows:
 
 | Name             | Description         | Default Value | Possible Data Types | Optional |
 |------------------|----------------------------------|---------------|---------------------|----------|
-| stream      | The streams to which the sink needs to publish events.|   | STRING        | No                  |
+| stream      | The streams to which the sink needs to publish events.|     | STRING        | No                  |
```

Comment:
@StoyanTc - The parameters list for stream-sink does not match the parameters list in stream.md, which I think is stream source. Is there an error here? What is going on?

Reply:
@diana-macrometa - Yes, the params list for source and sink streams are different.